perf/benchmark/configs/istio/plaintex_cpu_mem.yaml

```diff
@@ -0,0 +1,19 @@
+telemetry_mode: "none"
```

Comment:
what's the telemetry_mode you gonna name it? "none" here means nomixer.

Reply:
aha, you mean the hack in `run_benchmark_jobs.sh` to check the file prefix and then decide whether turning on mixer or not?

If so, I'll fix this, doing `grep on the config file`, instead of based on the prefix file name.

Reply:
@incfly if we need add a new line, then, we should have a new mode for this.

Reply:
discussed with @incfly, we can name this to be "none_plaintext"